contracts/connection-router/src/msg.rs

```diff
@@ -6,6 +6,7 @@ use crate::types::GatewayDirection;
 // Message is a type meant to be used in interfaces where the data can be provided by the user.
 // The fields have not necessarily been validated, and should be checked prior to further processing.
 #[cw_serde]
+#[deprecated(note = "use NewMessage instead")]
```

Comment:
yes. Of course, once the old one is deleted we can just call it `Message`

Reply:
I am a bit concerned with the ease of integration, since the types in this message are going to be used extensively by relayers. We would need to make sure to always serialize the types in a way that other languages can easily understand. If they always boil down to just a string or a byte array, that's fine. But we need to make sure that's the case, and document it clearly, etc.